.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: ci
+
+on:
+  push:
+    branches: [main, actions-test]
+  pull_request:
+    branches: [main, actions-test]
```

Comment:
```suggestion
on:
  pull_request:
  push:
    branches: main
```

Reply:
I think it would be nice to also have on pull requests